examples/wearable/UIComponents/index.html

```diff
@@ -30,6 +30,16 @@ <h2 class="ui-title">
 						Navigation elements
 					</a>
 				</li>
+				<li>
+					<a href="contents/content/index.html">
```

Comment:
Now contents/content folder is removed.
Do you have a plan to add the folder?

Reply:
No, it was added by accident. Thank you for your mark